src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Helpers/NativeMethods.cs

```diff
@@ -0,0 +1,1161 @@
+﻿// Copyright (c) Microsoft Corporation
```

Comment:
THEORETICALLY, I think we should be using NativeMethods.txt and let CsWin32 generate all this for us nowadays. 

But I'm not gonna ask you to do that by hand.

Reply:
roger that -- marking resolved.